packages/react/src/components/Tile/Tile-test.js

```diff
@@ -38,6 +38,13 @@ describe('Tile', () => {
     it('renders extra classes passed in via className', () => {
       expect(wrapper.hasClass('extra-class')).toEqual(true);
     });
+
+    it('supports light version', () => {
+      const wrapper = mount(<Tile>Test</Tile>);
+      expect(wrapper.props().light).toEqual(false);
+      wrapper.setProps({ light: true });
+      expect(wrapper.props().light).toEqual(true);
```

Comment:
should we also check that the `.bx--tile--light` class is applied?

Reply:
That's a good point. To be honest, I just copied the test over from `dropdown-test.js`.
I'll commit the extra test in a minute.

Reply:
That's a good point. To be honest, I just copied the test over from dropdown-test.js.
I'll commit the extra test in a minute.